cmd/registry-experimental/cmd/bleve/bleve_test.go

```diff
@@ -181,3 +191,83 @@ func TestSearch(t *testing.T) {
 		}
 	})
 }
+
+func TestServer(t *testing.T) {
+	ctx := context.Background()
+	blevePath := filepath.Join(t.TempDir(), "registry.bleve")
+	port := "8891"
+
+	buildTestRegistry(ctx, t)
+
+	// Start the server
+	go func() {
+		cmd := Command()
+		cmd.SetArgs([]string{"serve", "--bleve", blevePath, "--port", port})
+		if err := cmd.Execute(); err != nil {
+			log.Fatalf("Execute() with args %+v returned error: %s", cmd.Args, err)
+		}
+	}()
+
+	// Give it time to start.
+	time.Sleep(1 * time.Second)
```

Comment:
If possible, we should avoid non-deterministic stuff that could flake a test. I know we don't have a health check, but maybe we could just do `net.DialTimeout` in a bounded loop?

Reply:
OK, thanks! I'll fix that before submitting